subjects/java/checkpoints/valid-sudoku/README.md

```diff
@@ -0,0 +1,71 @@
+## Valid Sudoku
+
+### Instructions
+
+Determine if a 9x9 Sudoku board is valid. Only the filled cells need to be validated according to the following rules:
+
+1. Each row must contain the digits 1-9 without repetition.
+2. Each column must contain the digits 1-9 without repetition.
+3. Each of the nine 3x3 sub-boxes of the grid must contain the digits 1-9 without repetition.
```

Comment:
Nothing major, but it might be nice to add a picture to specify that the "sub-boxes" are the 9 highlighted in the picture ([this one](https://www.rd.com/wp-content/uploads/2020/12/Sudoku.jpg?fit=700,700) for example).

I guess sudoku is quite popular so people should know the rule therefore it might not be required.

Reply:
yeah as you say, it's popular so no need to add a picture to clarify.